cypress/e2e/notes-100/note-contains-link-to-github-issue.cy.js

```diff
@@ -19,7 +18,7 @@ describe('Notes 100: Select a note', () => {
       context('Open first note', () => {
         beforeEach(() => cy.get('[data-testid="list-notes"] :nth-child(1) > [data-testid="note-body"]').first().click())
```

Comment:
Here for example you added the list-notes and then note-body -- unless we are dynamically generating the test ID, i don't see the way to just look at the test id